### PR TITLE
S3 config changes for PRC RCA

### PR DESCRIPTION
**Pull request by venkatarmedia**
AWS S3 config changes for PRC RCA

### CODE REVIEW

File: vagrant/provisioning/arkcase-dev-facts.yml [hunk 1]
```diff
@@ -760,6 +760,8 @@ file_management_s3_buckets:
   - config_name: arkcaseBucket
     bucket_name: arkcase-portal
 
+prc_dockets_s3_bucket_name: prc-rca
```
Comment (david-oc-miller) [inline vagrant/provisioning/arkcase-dev-facts.yml:763]: What is the right bucket name for PRC dev?  I'll have to update the AWX Tower configuraton.

File: vagrant/provisioning/roles/arkcase-app/templates/arkcase-portal-prc-server.yaml [hunk 2]
```diff
@@ -2,7 +2,16 @@ portal-prc:
...
+    url: "{{ file_management_s3_url }}"
+    region: "{{ file_management_s3_region }}"
+    accessKey: "{{ file_management_s3_access_key }}"
+    secretKey: "{{ file_management_s3_secret_key }}"
+    bucketName: "{{ prc_dockets_s3_bucket_name }}"
+  
 arkcase.rca.solr.configuration:
   host: "{{ solr_host }}"
-  port: {{ solr_port }}
+  port: {{ solr_port | default(8983) }}
```
Comment (david-oc-miller) [inline vagrant/provisioning/roles/arkcase-app/templates/arkcase-portal-prc-server.yaml]: We found the default port should be 443 right?

Comment (venkatarmedia) [reply vagrant/provisioning/roles/arkcase-app/templates/arkcase-portal-prc-server.yaml]: yes, I looked at other places and it is 443

Comment (venkatarmedia) [reply vagrant/provisioning/roles/arkcase-app/templates/arkcase-portal-prc-server.yaml]: My bad, I made that change but didn't stage it.

File: vagrant/provisioning/arkcase-dev-facts.yml (hunk 1 above)
Comment (venkatarmedia) [reply vagrant/provisioning/arkcase-dev-facts.yml:763]: arkcase-fedramp-sandbox-file-management

**Review by david-oc-miller — APPROVED** [review]

**Comment by david-oc-miller** [comment]
PRC Dev AWX configuration updated, and the installer project updated in AWX.

**Pull request merged**